packages/create-hyperstack/template-app/src/config/environments/production.ts

```diff
@@ -63,6 +63,10 @@ export default async () => ({
     gzip: true,
     indexCatchAll: true,
     serveStatic: true,
+    bearer: {
+      header: 'x-axccess-token',
```

Comment:
similar typos below as well

Reply:
no just a small commit that fixes the typo would be good.
I'll be away from keyboard in the next few hours, but I can also do the fix myself if you prefer.

Reply:
Thanks, yes it makes sense.
I'll look at this as well.
It looks like a great contribution, I'll merge soon - thanks!